src/hooks/useVenueUserStatuses.ts

```diff
@@ -0,0 +1,32 @@
+import { DefaultUserStatus, User } from "types/User";
+
+import { USER_STATUSES } from "settings";
+import { WithId } from "utils/id";
+import { useRelatedVenues } from "./useRelatedVenues";
+import { useSovereignVenueId } from "./useSovereignVenueId";
```

Comment:
reorder the imports, please. 🙃 

```suggestion
import { USER_STATUSES } from "settings";

import { DefaultUserStatus, User } from "types/User";

import { WithId } from "utils/id";

import { useRelatedVenues } from "./useRelatedVenues";
import { useSovereignVenueId } from "./useSovereignVenueId";
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/2de383019a9ecbef3edfbcc744a06df2a0ffac64